File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        l = 0
        r = 0
        while r < len(nums):
            if val != nums[r]:
                nums[l] = nums[r]
                l += 1
            r += 1
        return l

File: Data Structures & Algorithms/jump-game/submission-0.cpp
class Solution {
public:
    bool canJump(vector<int>& nums) {
        int goal = nums.size() - 1;
        for (int i = nums.size() - 2; i >= 0; i--) {
            cout << goal << " " << i << " " << nums[i] << endl;
            if (goal - (i + nums[i]) <= 0) {
                goal = i;
            }
        }
        return goal == 0 ? true: false;
    }
};
